--- node_modules/interface-ipfs-core/src/refs-local.js
+++ node_modules/interface-ipfs-core/src/refs-local.js
@@ -48,10 +48,14 @@ module.exports = (common, options) => {

       const imported = await all(importer(dirs, ipfs.block))

-      // otherwise go-ipfs doesn't show them in the local refs
-      await Promise.all(
-        imported.map(i => ipfs.pin.add(i.cid))
-      )
+      // rust-ipfs doesn't yet have pinning api, it'll just list all local cids
+      // in /refs/local
+      if (common.opts.type !== 'rust') {
+        // otherwise go-ipfs doesn't show them in the local refs
+        await Promise.all(
+          imported.map(i => ipfs.pin.add(i.cid))
+        )
+      }

       const refs = await all(ipfs.refs.local())
       const cids = refs.map(r => r.ref)
--
2.27.0

